base/lib/migrate.mustache

```diff
@@ -11,7 +11,7 @@ const r = require('lib/db');
 const table = r.table('_migrations');
 
 const applyMigration = (migrations, then) => {
-  if (migrations.length === 0) return;
+  if (migrations.length === 0) return Promise.resolve();
```

Comment:
Yeah, that'll do. If we're already in Promise Land then it doesn't matter whether we return a resolve or a null, but it is probably better to return the resolve.